scripts/zones/The_Shrine_of_RuAvitau/mobs/Mother_Globe.lua

```diff
@@ -4,40 +4,109 @@
 -- TODO: Looked like pets had an additional effect: stun with an unknown proc rate
 -- TODO: "Links with Slave Globes, and Slave Globes link with Defenders. Defenders do not link with Slave Globes or Mother Globe."
 -----------------------------------
+local ID = require("scripts/zones/The_Shrine_of_RuAvitau/IDs")
 require("scripts/globals/status")
 -----------------------------------
 local entity = {}
 
+local findSlaveGlobeToSpawn = function()
+    local spawnCount = 0
+
+    for _, slaveGlobeID in ipairs(ID.mob.SLAVE_GLOBES) do
+        local slaveGlobe = GetMobByID(slaveGlobeID)
+
+        spawnCount = spawnCount + 1
+
+        if not slaveGlobe:isSpawned() then
+            return slaveGlobe, spawnCount
+        end
+    end
+
+    return nil, spawnCount
+end
+
+local calculateSlaveGlobePosition = function(mg, slaveGlobe, slavePositionSlot)
+    local mgPos = mg:getPos()
+    local xOffset = -1 * slavePositionSlot -- should offset based on the slot
+
+    return utils.lateralTranslateWithOriginRotation(mgPos, {x = xOffset, y = 0, z = 0})
+end
+
+local spawnSlaveGlobe = function(mg, slaveGlobe, slavePositionSlot)
+    local slaveGlobePos = calculateSlaveGlobePosition(mg, slaveGlobe, slavePositionSlot)
+    slaveGlobe:setSpawn(slaveGlobePos.x, slaveGlobePos.y, slaveGlobePos.z, slaveGlobePos.rot)
+    slaveGlobe:spawn()
+    slaveGlobe:setLocalVar("slavePositionSlot", slavePositionSlot)
+
+    if mg:isEngaged() then
+        slaveGlobe:updateEnmity(mg:getTarget())
+    end
+end
+
+local setNextSpawnSlaveGlobe = function(mg, spawnCount, nowTime)
+    local nextSpawnTime = 35 -- 30 + 5 seconds for "cast time"
+
+    if spawnCount < #ID.mob.SLAVE_GLOBES then
+        mg:setLocalVar("nextSlaveSpawnTime", nowTime + nextSpawnTime)
+    else
+        mg:setLocalVar("nextSlaveSpawnTime", 0)
+    end
+end
+
+local trySpawnSlaveGlobe = function(mg, nowTime)
+    local nextSlaveSpawnTime = mg:getLocalVar("nextSlaveSpawnTime")
+    local shouldTryToSummonSlaveGlobe = nowTime > nextSlaveSpawnTime
+    local inCombat = mg:isEngaged()
+    local combatHasNotRecentlyStarted = mg:getBattleTime() > 3
+
+    if shouldTryToSummonSlaveGlobe and (not inCombat or combatHasNotRecentlyStarted) then
+        local slaveGlobe, spawnCount = findSlaveGlobeToSpawn(mg)
+        if slaveGlobe then
+            spawnSlaveGlobe(mg, slaveGlobe, spawnCount)
+            setNextSpawnSlaveGlobe(mg, spawnCount, os.time())
+        end
+    end
+end
+
+local handleSlaveGlobesRoam = function(mg)
+    local mgPos = mg:getPos()
+
+    for _, slaveGlobeID in ipairs(ID.mob.SLAVE_GLOBES) do
+        local slaveGlobe = GetMobByID(slaveGlobeID)
+
+        if slaveGlobe:isSpawned() then
+            local slavePositionSlot = slaveGlobe:getLocalVar("slavePositionSlot")
```

Comment:
missed this in my refactor, should be using the calculateSlaveGlobePosition() method